clip2onnx/model_card.py

```diff
@@ -0,0 +1,165 @@
+import inspect
+import logging
+import sys
+import warnings
+from dataclasses import dataclass, field
+from pathlib import Path
+
+import clip
+import open_clip
+import torch
+from PIL import Image
+
+warnings.filterwarnings("ignore", category=UserWarning)
+
+logging.basicConfig(
+    level=logging.INFO, stream=sys.stdout, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+
+@dataclass
+class ModelCard:
```

Comment:
`ModelCard` seems to imply that this is just information about the model, but it's actually also meant for downloading it. Maybe just call this `ClipModel`?